voluncheer/voluncheer/settings.py

```diff
@@ -91,6 +94,29 @@
 ]
 
 WSGI_APPLICATION = "voluncheer.wsgi.application"
+ASGI_APPLICATION = "voluncheer.asgi.application"
+
+
+REDIS_CHATROOM_PORT = os.getenv("REDIS_CHATROOM_PORT")
+if environment.is_production or environment.is_development:
```

Comment:
We have the `environment.is_aws` which obfuscates away the specific environment.

https://github.com/gcivil-nyu-org/INET-Monday-Spring2023-Team-2/blob/develop/voluncheer/voluncheer/environment.py#L24-L26